docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+services:
+  ow_gym_sim:
```

Comment:
Place comment that this is only used for build purposes, please get rid of build, volumes, networks, ports tag

Reply:
DOne